src/cascadia/TerminalSettingsEditor/SettingContainer.cpp

```diff
@@ -133,43 +133,46 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         _UpdateOverrideSystem();
 
         // Get the correct base to apply automation properties to
-        DependencyObject base{ nullptr };
+        std::vector<DependencyObject> base(2, nullptr);

```

Comment:
wait, doesn't this create a vector with `nullptr, nullptr` and then you add two _new_ elements to it (`nullptr, nullptr, expander, content`)?

Recommend .. not doing that. I saw the nullptr check down on R154 and it made me realize something was amiss. You're never adding a nullptr, so why check?